### PR TITLE
Hotfixes loadout cloak path

### PR DESCRIPTION
**Pull request by NopemanMcHalt**
Changes cloak path, so jobwise cloaks wont get gear selection for no reason.

I'm complete buffoon.

### CODE REVIEW

**Comment by DogZeroX** [comment]
I'm the local cheese.

File: code/modules/clothing/cloak/hooded.dm [hunk 1]
```diff
@@ -93,7 +93,7 @@
 	name = "cloak hood"
 	icon_state = "cloak_hood"
 	desc = "A hood from cloak. It hides away your eyes and head."
-	flags_inv = HIDEEARS|HIDEEYES|BLOCKHEADHAIR
```
Comment (DogZeroX) [inline code/modules/clothing/cloak/hooded.dm]: Why don't just add a verb options like **Change-Ears-Layer**

Comment (king5327) [reply code/modules/clothing/cloak/hooded.dm]: So we're confirming that the cloak, something which normally would hide ears, doesn't in this case?

Comment (NopemanMcHalt) [reply code/modules/clothing/cloak/hooded.dm]: I reverted it. Forcefully.

If custom ears are going to be getting removed again then I'm not sure what to do. And I'm too incompetent for making new verbs with new defines and shit.

**Comment by APWill** [comment]
Closing due to conflict with the big clothing PR which fixes this issue. Sorry. I got merge access here today so I'll be able to quickly approve any useful changes like this in future. : )

**Pull request closed without merging**